test/sly-package-fu-tests.el

```diff
@@ -0,0 +1,54 @@
+;; -*- lexical-binding: t; -*-
+(require 'sly-package-fu "contrib/sly-package-fu")
+(require 'sly-tests "lib/sly-tests")
+
+(def-sly-test sly-package-fu-sly-import (initial-defpackage final-defpackage symbol)
+              "Check if importing `import` on `initial-defpackage` results in `final-defpackage."
+              '(("(defpackage :foo
+  (:use :cl))
+(in-package :foo)"
+                 "(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:find))
+(in-package :foo)"
+                 "cl:find")
+                ("(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:find))
+(in-package :foo)"
+                 "(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:position
+                #:find))
+(in-package :foo)"
+                 "cl:position")
+
+                ;; TODO: the output here is incorrect, but we're
+                ;; documenting the current behaviour in this test.
+                ("(defpackage :foo
+  (:use :cl)
+  (:import-from #:bknr.datastore
```

Comment:
Are you sure this test runs without `bknr.datastore` loaded into the lisp?

Reply:
It seemed to pass. I just assumed I used a low enough function that didn't make a call into the image, but let me double check that it's not pulling something from my .sbclrc.

Reply:
I've replaced this with bknr.datastore-dummy just to emphasize that it doesn't depend on bknr.datastore being loaded. The tests still pass.